app/2.0.x/db-less-admin-api.md

```diff
@@ -562,11 +569,45 @@ of methods to secure the Admin API.
 
 ## Supported Content Types
 
-The Admin API accepts 2 content types on every endpoint:
+The Admin API accepts 3 content types on every endpoint:
```

Comment:
This change existed on admin-api.md but not here. I have updated the Kong data so it includes the 3 content types. Re-generating the template updated this section on `db-less-admin-api.md`.